code/globalincs/scp_defines.h

```diff
@@ -6,4 +6,7 @@
 #include "scp_compiler_detection.h"
 #include "platformChecks.h"
 
+#define SCP_TOKEN_CONCAT1(x, y) x ## y
+#define SCP_TOKEN_CONCAT(x, y) SCP_TOKEN_CONCAT1(x, y)
```

Comment:
This is the usual way concatenation is implemented using C preprocessor.
IIRC If you only use 1 you can't give macros as arguments